hrjobcontract/tests/phpunit/CRM/Hrjobcontract/BAO/HRJobContractRevisionTest.php

```diff
@@ -19,43 +28,18 @@ public function setUpHeadless() {
   }
 
   function testFullDetailsResponse() {
```

Comment:
Since this is testing an API method , I think it should be moved to another folder (**tests/phpunit/api/v3/HRJobContractTest.php** )